front/lib/resources/tracker_resource.ts

```diff
@@ -264,6 +271,67 @@ export class TrackerConfigurationResource extends ResourceWithSpace<TrackerConfi
     });
   }
 
+  static async fetchWithGenerationsToConsume(
```

Comment:
I decided to not fetch those by default in the baseFetch because we only needs the generations on this specific workflow, never on the app.